python/triton/language/core.py

```diff
@@ -1179,7 +1179,7 @@ def arange(start, end, _builder=None):
     """
     Returns contiguous values within the half-open interval :code:`[start,
     end)`.  :code:`end - start` must be less than or equal to
-    :code:`TRITON_MAX_TENSOR_NUMEL = 131072`
+    :code:`TRITON_MAX_TENSOR_NUMEL`
```

Comment:
I meant something like:
```
:code:`TRITON_MAX_TENSOR_NUMEL = {TRITON_MAX_TENSOR_NUMEL}`
```

Reply:
Thanks for the suggestion. Was looking for an example to evaluate the variable in doc... Updated accordingly.

Reply:
I think you need an extra `f` at the beginning of the string

Reply:
Many thanks for the patient guide... 
(Im more familiar with the Pytorch doc , e.g., https://github.com/pytorch/pytorch/blob/main/torch/_tensor_docs.py and the Triton doc seems a bit different than the Pytorch ones.)